### DISCUSSION TITLE
Feedback: JUnit reports

### DISCUSSION THREAD

**Discussion by Stranger6667** [schemathesis/schemathesis#2060] [Ideas]
As of now, JUnit reports generated by Schemathesis don't provide enough useful information. Lets improve it!

What kind of information would you expect in JUnit reports?

One of the goals I have is to have a convenient & informative test run report in e.g. Jenkins (and anything else that supports JUnit), so any screenshots are welcome.

P.S. Additionally I want to switch to a different library for writing such reports as the one we use wasn't updated for quite some time

**Comment by adnweedon** [comment]
I'll start the batting!

This was a recent test failure (as visualised in Semaphore's UI):
<img width="875" alt="image" src="https://github.com/schemathesis/schemathesis/assets/81413869/39868be7-959a-4032-9b96-0eee80dcea87">

It would be really helpful if the stdout information could be seen here, which is much more helpful in actually diagnosing the problem, without having to dig through the logs:

<img width="604" alt="image" src="https://github.com/schemathesis/schemathesis/assets/81413869/c26bd833-5384-4b4a-ae08-649c1696a978">


Hypothesis errors come through in a helpful format already though!
![image](https://github.com/schemathesis/schemathesis/assets/81413869/5e56aaa7-cc9c-4143-8baa-2a653bf46e06)

**Reply by Stranger6667** [reply]
I hope to finish https://github.com/schemathesis/schemathesis/pull/2132 over this weekend and release it in a new minor version :) The JUnit output will be close to CLI.

If you'd have any additional ideas or concerns, please let me know
